src/tests/interface/сommands_interface/commands/history/HistoryPaymentsCommandTest.cpp

```diff
@@ -55,20 +96,26 @@ TEST_CASE("Testing HistoryPaymentsCommand")
         REQUIRE_THROWS(HistoryPaymentsCommand("47183823-2574-4bfd-b411-99ed177d3e43"s, "1\t2\t3\t4\t5\t6\t47183823-25574-4bfd-b411-99ed177dd3e43\t8\n"));
     }
 
-    SECTION("Max number in input")
+    SECTION("Max number in input (77 digits)")
     {
-        REQUIRE_NOTHROW(HistoryPaymentsCommand("47183823-2574-4bfd-b411-99ed177d3e43"s,"1\t2\t3\t4\t340282366920938463463374607431768211456\t6\t47183823-2574-4bfd-b411-99ed177dd3e4\t8\n"));
+        REQUIRE_NOTHROW(HistoryPaymentsCommand("47183823-2574-4bfd-b411-99ed177d3e43"s,"1\t2\t3\t4\t115792089237316195423570985008687907853269984665640564039457584007913129639935\t6\t47183823-2574-4bfd-b411-99ed177dd3e4\t8\n"));
 
-        REQUIRE_NOTHROW(HistoryPaymentsCommand("47183823-2574-4bfd-b411-99ed177d3e43"s,"1\t2\t3\t4\t5\t340282366920938463463374607431768211456\t47183823-2574-4bfd-b411-99ed177de435\t8\n"));
+        REQUIRE_NOTHROW(HistoryPaymentsCommand("47183823-2574-4bfd-b411-99ed177d3e43"s,"1\t2\t3\t4\t5\t115792089237316195423570985008687907853269984665640564039457584007913129639935\t47183823-2574-4bfd-b411-99ed177de435\t8\n"));
```

Comment:
Have you tried 115792089237316195423570985008687907853269984665640564039457584007913129639936 (last digit is 6 instead of 5)?